### ISSUE TITLE
Explore incorporatating softcite/softcite_kb data into Augur for Academic Metrics

### ISSUE THREAD

**Issue by sgoggins** [chaoss/augur#2292]
**Is your feature request related to a problem? If so, please describe the problem:**
Working in the context of the @chaoss project, we are developing metrics for Academic open source contexts. These include alt metrics related to software, as well as more conventional metrics related to academic publications. 

https://github.com/softcite/softcite_kb is a project that could help support this effort.

**Comment by jmelot** [comment]
I'm working on a (somewhat) related project and would very much like to help out with this as I have spare time. If you're interested in contributions, could you say more about what a good starting point would be here?

**Comment by Madhu-T18** [comment]
@sgoggins can you assign for me?

**Comment by MoralCode** [comment]
@sgoggins can you clarify how you see this integrating with Augur? Are you seeing this as being a useful tool for text mining github issues and comments? or are you thinking more along the lines of expanding augur to also gather data on academic works associated with source code repos? or something else?


Is there a specific CHAOSS metric this is working towards as well?

also given that this has good first issue tags, it may be worth clarifying whether the intended scope is limited to exploration, not implementation (at this stage) - i.e. are we just trying to gather options and ideas for how this could work to later evaluate and decide which to implement?

Are we worried about such a first-timers-only research task leading to AI-generated submissions that cost us more time than if we simply did the research ourselves?

**Comment by Madhu-T18** [comment]
Yes, sir my intention is to stay within Augur’s existing boundaries and
collaborate with the team to improve insights derived from GitHub issues
and comments that Augur already collects, without introducing new data
sources.
